[Content] Coalition Megastructures (#6945)
endless-sky/endless-sky: data/map.txt

		threshold 3500
		fleet "Large Syndicate" 7

planet "Garden Empyreal"
	landscape

> 	description `Not counting the trio of Quarg ringworlds, this is the largest megastructure in Coalition space, an array of gigantic metal pyramids, each one nearly a dozen times larger than the average space station. Spread out over the planet's upper atmosphere, these so-called "hubs" are connected by long, inextensible, and extremely sturdy wires, ensuring no part of the structure is destabilized.`
> 	description `	First devised by the Kimek before the Coalition's war with the Quarg, the structure was actually completed relatively recently, just two thousand years ago. Populating the gas giant's uppermost level, several hardy species of artificially engineered bacteria are cultivated, being periodically "scooped" by the lower body of each hub, and mass-produced into a nutritious soup.`

Reviewer: ```suggestion
	description `Not counting the ringworlds, this is the largest megastructure in Coalition space, an array of gigantic metal pyramids, each one nearly a dozen times larger than the average space station. Spread out over the planet's upper atmosphere, these "hubs" are connected by long, inextensible, and extremely sturdy wires, ensuring no part of the structure is destabilized.`
	description `	Devised by the Kimek before the Coalition's war with the Quarg, it was actually completed relatively recently, two thousand years ago. Populating the gas giant's uppermost level, several hardy species of artificially engineered bacteria are cultivated, being periodically "scooped" by the lower body of each hub, and processed into a nutritious soup.`
```

		threshold 3500
		fleet "Large Syndicate" 7

> planet "Garden Empyreal"
> 	landscape

Reviewer: ```suggestion
planet "Garden Empyreal"
	attributes kimek "coalition station" shipping wealthy tourism
	landscape
```

	description `	First devised by the Kimek before the Coalition's war with the Quarg, the structure was actually completed relatively recently, just two thousand years ago. Populating the gas giant's uppermost level, several hardy species of artificially engineered bacteria are cultivated, being periodically "scooped" by the lower body of each hub, and mass-produced into a nutritious soup.`
	spaceport `When not in "harvest season", the hubs are relatively quiet, and serve mostly as a tourist destination, many coming for the unique, close-up view of a sea of clouds, swarming with furious storms and dotted with more floating silver palaces in the distance.`
	spaceport `	Hubs share the same design in general, but the decoration pieces, services and local amenities of each one have grown in their own way over the centuries. The only shop present in every one of them is the signature restaurant, which serves a variety of dishes, drinks and even desserts all based on the processed, scooped up produce.`

> 	"required reputation" 15

Reviewer: ```suggestion
	"required reputation" 20
```

> 	"required reputation" 15

Reviewer: ```suggestion
	"required reputation" 20
```

		threshold 3500
		fleet "Large Syndicate" 7

planet "Garden Empyreal"
	attributes kimek "coalition station" shipping wealthy tourism

> 	landscape

Reviewer: ```suggestion
	landscape land/clouds_03
```

> 	description `Not counting the trio of Quarg ringworlds, this is the largest megastructure in Coalition space: an array of gigantic metal pyramids, each one nearly a dozen times larger than the average space station. Spread out over the planet's upper atmosphere, these so-called "hubs" are connected by long, inextensible, and extremely sturdy wires, ensuring no part of the structure is destabilized.`

Reviewer 2: ```suggestion
	description `Not including the trio of Quarg ringworlds, this is the largest megastructure in Coalition space. Arrays of gigantic metal pyramids lay at every angle, each nearly a dozen times larger than the average space station. Spread out over the planet's upper atmosphere, these so-called "hubs" are connected by long, inextensible, and extremely sturdy wires, ensuring no part of the structure is destabilized.`
```

> 	description `	First devised by the Kimek before the Coalition's war with the Quarg, the structure was actually completed relatively recently, just two thousand years ago. Populating the gas giant's uppermost level, several hardy species of artificially engineered bacteria are cultivated, being periodically "scooped" by the lower body of each hub, and mass-produced into a nutritious soup.`

Reviewer 2: ```suggestion
	description `	First devised by the Kimek before the Coalition's war against the Quarg, the structure was completed relatively recently two thousand years ago. Populating the gas giant's uppermost atmosphere laid several hardy species of artificially  engineered bacteria, periodically being "scooped" by the lower body of each hub and then later mass-produced into a nutritious soup.`
```

> 	description `	First devised by the Kimek before the Coalition's war against the Quarg, the structure was completed relatively recently two thousand years ago. Populating the gas giant's uppermost atmosphere laid several hardy species of artificially  engineered bacteria, periodically being "scooped" by the lower body of each hub and then later mass-produced into a nutritious soup.`

Reviewer 3: ```suggestion
	description `	First devised by the Kimek before the Coalition's war against the Quarg, the structure was completed relatively recently two thousand years ago. Populating the gas giant's uppermost atmosphere lay several hardy species of artificially engineered bacteria, periodically being "scooped" by the lower body of each hub and then later mass-produced into a nutritious soup.`
```

> 	spaceport `When not in "harvest season", the hubs are relatively quiet, and serve mostly as a tourist destination. Many come for the unique, close-up view of a sea of clouds, swarming with furious storms and dotted with more floating silver palaces in the distance.`

Reviewer: "floating silver palaces" is supposed to mean the pyramid hubs.

> 	spaceport `When not in "harvest season", the hubs are relatively quiet, and serve mostly as a tourist destination. Many come for the unique, close-up view of a sea of clouds, swarming with furious storms and dotted with more floating silver palaces in the distance.`

Reviewer 2: Ah, ok...
Might want to address it so that it makes the connection a bit more obvious, something like referencing the triangle-ness of  each hub, or reflection, or something close to that.

> 	spaceport `When not in "harvest season", the hubs are relatively quiet, and serve mostly as a tourist destination. Many come for the unique, close-up view of a sea of clouds, swarming with furious storms and dotted with more floating silver palaces in the distance.`

Reviewer 2: Better. Yep.

> 	spaceport `When not in "harvest season", the hubs are relatively quiet, and serve mostly as a tourist destination, many coming for the unique, close-up view of a sea of clouds, swarming with furious storms and dotted with more floating silver palaces in the distance.`

Reviewer 4: ```suggestion
	spaceport `When not in "harvest season", the hubs are relatively quiet, and serve mostly as a tourist destination. Many come for the unique, close-up view of a sea of clouds, swarming with furious storms and dotted with more floating silver palaces in the distance.`
```

> 	spaceport `When not in "harvest season", the hubs are relatively quiet, and serve mostly as a tourist destination. Many come for the unique, close-up view of a sea of clouds, swarming with furious storms and dotted with more of the silver pyramids in the distance.`

Reviewer 3: ```suggestion
	spaceport `When not in "harvest season," the hubs are relatively quiet, and serve mostly as a tourist destination. Many come for the unique, close-up view of a sea of clouds, swarming with furious storms and dotted with more of the silver pyramids in the distance.`
```

> 	spaceport `	Each hub shares the same overall design in general, but the decoration pieces, services, and local amenities for each one have grown in their own way over the centuries. The only common thing they have between all of them is the signature restaurant, which serves a variety of dishes, drinks and even desserts all based on the processed, scooped up produce.`

Reviewer 3: ```suggestion
	spaceport `	Each hub shares the same overall design in general, but the decoration pieces, services, and local amenities for each one have grown in their own way over the centuries. The only common thing they have between all of them is the signature restaurant, which serves a variety of dishes, drinks, and even desserts all based on the processed, scooped up produce.`
```

	bribe 0.1
	security 0.5

planet "Guardian Array Sapphire"
	description `More commonly referred to by its nickname "Checkpoint of Ablodab," this enormous station was built shortly after the end of the Coalition's war with the Quarg, meant to serve as a regional dry dock that damaged combat ships could return to after some time in the front lines. With no Quarg attack in so long, though, it's shifted into being a simple refuelling hub for passingby merchant ships, as its dozens of warship-sized hangars remain mostly unused.`

> 	description `	In spite of the lack of combat activity the station was designed to support, the local Heliarchs run a tight schedule even for civilian visitors as they first land, ensuring ships are refuelled and tended to as quickly as possible. On scheduled days of each month, Heliarch fleets that were damaged in training drills against one another come in for the station's repair capabilities to be put to a stress test.`
> 	spaceport `Built to serve as a purely strategical military repair port, the station has very little in the form of amenities, with a few basic cantinas and large, mostly empty living quarters, meant to house troops stationed here. There are no shops, and nearly all civilians you see are moving from their spaceships, on their way to a quick break, or heading back, ready to depart and continue their business.`

Reviewer: ```suggestion
	description `	In spite of the lack of combat activity the station was designed to support, the local Heliarchs run a tight schedule even for civilian visitors as they first land, ensuring ships are refuelled and tended to as quickly as possible.`
	spaceport `Built to serve as a purely strategical military repair port, the station has very little in the form of amenities, with a few basic cantinas and large, mostly empty living quarters, meant to house troops stationed here. There are no shops, and nearly all civilians you see are moving from their spaceships, on their way to a quick break, or heading back, ready to depart and continue their business.`
	spaceport `	On scheduled days of each month, Heliarch fleets that were damaged in training drills come in to stress test the station's repair capabilities, leading to a sudden boom in the station's population, and a crowding of the facilities..`
```

	bribe 0.1
	security 0.5

> planet "Guardian Array Sapphire"
> 	landscape

Reviewer: ```suggestion
planet "Guardian Array Sapphire"
	attributes heliarch arach "coalition station" 
	landscape
```

	description `	In spite of the lack of combat activity the station was designed to support, the local Heliarchs run a tight schedule even for civilian visitors as they first land, ensuring ships are refuelled and tended to as quickly as possible.`
	spaceport `Built to serve as a purely strategical military repair port, the station has very little in the form of amenities, with a few basic cantinas and large, mostly empty living quarters, meant to house troops stationed here. There are no shops, and nearly all civilians you see are moving from their spaceships, on their way to a quick break, or heading back, ready to depart and continue their business.`
	spaceport `	On scheduled days of each month, Heliarch fleets that were damaged in training drills come in to stress test the station's repair capabilities, leading to a sudden boom in the station's population, and a crowding of the facilities..`

> 	"required reputation" 15

Reviewer: ```suggestion
	"required reputation" 25
	security 0.6
```

	bribe 0.1
	security 0.5

planet "Guardian Array Sapphire"
	attributes heliarch arach "coalition station" 

> 	landscape

Reviewer: ```suggestion
	landscape land/station33
```

> 	description `More commonly referred to by its nickname "Checkpoint of Ablodab," this enormous station was built shortly after the end of the Coalition's war with the Quarg, meant to serve as a regional dry dock that damaged combat ships could return to after some time in the front lines. With no Quarg attack in so long, though, it's shifted into being a simple refuelling hub for passing merchant ships, as its dozens of warship-sized hangars remain mostly unused.`

Reviewer 2: ```suggestion
	description `More commonly referred to as "Checkpoint of Ablodab," this enormous station was built shortly after the end of the Coalition's war against the Quarg, meant to serve as a regional dry dock and supply base that damaged combat ships could return to after some time at the front lines. The lack of any subsequent counterattack, though, shifted its role to being a simple refuelling hub for passing merchant ships, its dozens of warship-sized hangars remain mostly unused.`
```

> 	description `More commonly referred to by its nickname "Checkpoint of Ablodab," this enormous station was built shortly after the end of the Coalition's war with the Quarg, meant to serve as a regional dry dock that damaged combat ships could return to after some time in the front lines. With no Quarg attack in so long, though, it's shifted into being a simple refuelling hub for passingby merchant ships, as its dozens of warship-sized hangars remain mostly unused.`

Reviewer 4: ```suggestion
	description `More commonly referred to by its nickname "Checkpoint of Ablodab," this enormous station was built shortly after the end of the Coalition's war with the Quarg, meant to serve as a regional dry dock that damaged combat ships could return to after some time in the front lines. With no Quarg attack in so long, though, it's shifted into being a simple refuelling hub for passing merchant ships, as its dozens of warship-sized hangars remain mostly unused.`
```

> 	description `More commonly referred to as "Checkpoint of Ablodab," this enormous station was built shortly after the end of the Coalition's war against the Quarg, meant to serve as a regional dry dock and supply base that damaged combat ships could return to after some time at the front lines. The lack of any subsequent counterattack, though, shifted its role to being a simple refuelling hub for passing merchant ships, its dozens of warship-sized hangars remain mostly unused.`

Reviewer 3: ```suggestion
	description `More commonly referred to as "Checkpoint of Ablodab," this enormous station was built shortly after the end of the Coalition's war against the Quarg, meant to serve as a regional dry dock and supply base that damaged combat ships could return to after some time at the front lines. The lack of any subsequent counterattack, though, shifted its role to being a simple refueling hub for passing merchant ships, its dozens of warship-sized hangars remaining mostly unused.`
```

> 	description `	In spite of the lack of combat activity the station was designed to support, the local Heliarchs run a tight schedule even for civilian visitors as they first land, ensuring ships are refuelled and tended to as quickly as possible.`

Reviewer 2: ```suggestion
	description `	Despite the apparent lack of combat activity the station was designed to support, the local Heliarchs run a tight schedule even for civilian visitors as they land, ensuring ships are refuelled and tended to as quickly as possible.`
```

> 	spaceport `Built to serve as a purely strategic military repair port, the station has very little in the form of amenities, with a few basic cantinas and large, mostly empty living quarters, meant to house troops stationed here. There are no shops, and nearly all civilians you see are moving from their spaceships on their way to a quick break, or heading back ready to depart and continue their business.`

Reviewer 4: ```suggestion
	spaceport `Built to serve as a purely strategic military repair port, the station has very little in the form of amenities, with a few basic cantinas and large, mostly empty living quarters meant to house troops stationed here. There are no shops, and nearly all civilians you see are moving from their spaceships on their way to a quick break, or heading back ready to depart and continue their business.`
```

> 	spaceport `Built to serve as a purely strategical military repair port, the station has very little in the form of amenities, with a few basic cantinas and large, mostly empty living quarters, meant to house troops stationed here. There are no shops, and nearly all civilians you see are moving from their spaceships, on their way to a quick break, or heading back, ready to depart and continue their business.`

Reviewer 4: ```suggestion
	spaceport `Built to serve as a purely strategic military repair port, the station has very little in the form of amenities, with a few basic cantinas and large, mostly empty living quarters, meant to house troops stationed here. There are no shops, and nearly all civilians you see are moving from their spaceships on their way to a quick break, or heading back ready to depart and continue their business.`
```

> 	spaceport `	On scheduled days of each month, Heliarch fleets that were damaged in training drills come in to stress test the station's repair capabilities, leading to a sudden boom in the station's population, and a crowding of the facilities...`

Reviewer 2: ```suggestion
	spaceport `	On certain days every month, Heliarch ships "damaged" during combat exercises dock to stress test the station's turnover capabilities, breathing new life in the station's population, and extreme overcrowding of the facilities...`
```

Reviewer 2: I've never been sent overseas for exercises (Not yet my time to do so,) but my understanding is this. Battalion level exercises, especially for logistics, are a massive undertaking, neverminding the fact that you have to settle food for Muslims, make sure every piece of equipment is indented properly, and the minor stuff that can easily get you extra duties.

But for the most part of the year, these bases aren't fully staffed, so when you have a battalion, or even a brigade doing exercises, things are hectic, and most likely, shit has already hit the fan.

 The requested edits for this one is intended to replicate this in writing, so if you don't think it fits, contact me. I may or may not be able to answer depending on how busy I am.

Reviewer: I don't mind the edits, can commit these.

> 	spaceport `	On scheduled days of each month, Heliarch fleets that were damaged in training drills come in to stress test the station's repair capabilities, leading to a sudden boom in the station's population, and a crowding of the facilities..`

Reviewer 4: ```suggestion
	spaceport `	On scheduled days of each month, Heliarch fleets that were damaged in training drills come in to stress test the station's repair capabilities, leading to a sudden boom in the station's population, and a crowding of the facilities...`
```

> 	spaceport `Built to serve as a purely strategic military repair port, the station has very little in the form of amenities, with a few basic cantinas and large, mostly empty living quarters intended to house troops stationed here. There are no shops, and nearly every civilian you see is moving from their spaceships on their way to a quick break, or heading back, ready to depart and continue their business.`
> 	spaceport `	On certain days every month, Heliarch ships "damaged" during combat exercises dock to stress test the station's turnover capabilities, breathing new life in the station's population, and extreme overcrowding of the facilities...`

Reviewer 3: ```suggestion
	spaceport `Built to serve as a purely strategic military repair port, the station has very little in the form of amenities, with a few basic cantinas and large, mostly empty living quarters intended to house troops stationed here. There are no shops, and nearly every civilian you see is moving as if they are trying to finish their business on the station as quickly as possible.`
	spaceport `	On certain days every month, Heliarch ships "damaged" during combat exercises dock to stress test the station's turnover capabilities, breathing new life into the station's population, and causing extreme overcrowding of the facilities...`
```

	bribe 0.1
	security 0.5

planet "Ogmur's Siphon"

> 	description `This refinery was built shortly after the nearby dry dock station was finished, to ensure the system was self-sufficient in terms of hyperspace fuel. With the exclusive goal of serving as a large scale refinery, the station has far less living grounds than the system's main attraction, often leading to semi-crowded areas in spite of the low amount of people here. A small fleet of tanker ships are docked on specially marked hangar bays, always ready to bring the fuel elsewhere if needed.`
> 	description `	"Ogmur" is what the Arachi baptized the gas giant below, naming it after a giant snake-like creature of their mythology. With an abnormally dense core, and storms more ruthless than in any other gas giant in the Coalition, Ogmur swallowed the first five iterations of this station before they could be completed, and though this one has remained stable for thousands of years after its completion, superstition surrounding the planet makes it so the Siphon gets very few visitors.`
> 	spaceport `While technically part of the system's "Guardian" military complex, there's little Heliarch control over the free space in this station, so a few shops and brand restaurants have opened up to serve the local workers and occasional visitor. Whether by necessity due to the lack of services in the system, or simply boredom in the monotony of the station, the food here is by far the best in the system, and even better than in some Coalition planets.`

Reviewer: ```suggestion
	description `This refinery was built shortly after the nearby dry dock station was finished, to ensure the system was self-sufficient in terms of hyperspace fuel. A small fleet of tanker ships are docked on specially marked hangar bays, always ready to bring the fuel elsewhere if needed.`
	description `	"Ogmur" is what the Arachi baptized the gas giant below, naming it after a giant snake-like creature of their mythology. With an abnormally dense core, and storms more ruthless than in any other gas giant in the Coalition, Ogmur swallowed the first five iterations of this station before they could be completed, and though this one has remained stable for thousands of years after its completion, superstition surrounding the planet makes it so the Siphon gets very few visitors.`
	spaceport `While technically part of the system's "Guardian" military complex, there's little Heliarch control over the free space in this station, so a few shops and brand restaurants have opened up to serve the local workers and occasional visitor. Whether by necessity due to the lack of services in the system, or simply boredom in the monotony of the station, the food here is by far the best in the system, and even better than in some Coalition planets.`
	spaceport `	With the exclusive goal of serving as a large scale refinery, the station has far less living grounds than the system's main attraction, often leading to semi-crowded areas in spite of the low amount of people here.`
```

	bribe 0.1
	security 0.5

> planet "Ogmur's Siphon"
> 	landscape

Reviewer: ```suggestion
planet "Ogmur's Siphon"
	attributes arach "coalition station" research
	landscape
```

	bribe 0.1
	security 0.5

planet "Ogmur's Siphon"
	attributes arach "coalition station" research

> 	landscape

Reviewer: ```suggestion
	landscape land/station36
```

	description `This refinery was built shortly after the nearby dry dock station was finished, to ensure the system was self-sufficient in terms of hyperspace fuel. A small fleet of tanker ships are docked on specially marked hangar bays, always ready to bring the fuel elsewhere if needed.`

> 	description `	"Ogmur" is what the Arachi baptized the gas giant below, naming it after a giant snake-like creature of their mythology. With an abnormally dense core, and storms more ruthless than in any other gas giant in the Coalition, Ogmur swallowed the first five iterations of this station before they could be completed, and though this one has remained stable for thousands of years after its completion, superstition surrounding the planet makes it so the Siphon gets very few visitors.`

Reviewer 2: ```suggestion
	description `	"Ogmur" is what the Arachi baptized the gas giant below, named after a giant snake-like creature of their mythology. With an abnormally dense core and storms more ruthless than in any other gas giants in the Coalition, it swallowed the first five predecessors of this station before they could be completed, and though this one has remained in a stable orbit for thousands of years after its completion, superstition surrounding the planet makes it so the Siphon receives very few visitors.`
```

> 	description `	"Ogmur" is what the Arachi baptized the gas giant below, named after a giant snake-like creature of their mythology. With an abnormally dense core and storms more ruthless than in any other gas giants in the Coalition, it swallowed the first five predecessors of this station before they could be completed, and though this one has remained in a stable orbit for thousands of years after its completion, superstition surrounding the planet makes it so the Siphon receives very few visitors.`

Reviewer 3: ```suggestion
	description `	"Ogmur" is what the Arachi baptized the gas giant below, named after a giant snake-like creature of their mythology. With an abnormally dense core and storms more ruthless than in any other gas giants in the Coalition, it swallowed the first five predecessors of this station before they could be completed, and though this one has remained in a stable orbit for thousands of years after its completion, superstition surrounding the planet makes it so that the Siphon receives very few visitors.`
```

> 	spaceport `While technically part of the system's "Guardian" military complex, there's little Heliarch control over the free space in this station, so a few shops and brand restaurants have opened up to serve the local workers and occasional visitor. Whether by necessity due to the lack of services in the system, or simply boredom in the monotony of the station, the food here is by far the best in the system, and even better than in some Coalition planets.`

Reviewer 2: ```suggestion
	spaceport `While technically part of the system's "Guardian" military complex, there's little to no Heliarch control over the free space in this station, so a few shops and branded restaurants have opened up to serve the local workers and occasional visitors. Whether by necessity due to the lack of services in the system, or simply out of boredom within the monotony of the station, the food here is by far the best in the system and even better than in some Coalition planets.`
```

> 	spaceport `	With the exclusive goal of serving as a large scale refinery, the station has far less living grounds than the system's main attraction, often leading to semi-crowded areas in spite of the low amount of people here.`

Reviewer 2: ```suggestion
	spaceport `	With the exclusive goal of serving as a large-scale refinery, the station has far less living space than the system's main attraction, often leading to semi-crowded areas in spite of the low amount of pedestrian traffic here.`
```

	bribe 0.1
	security 0.5

planet "Ogmur's Siphon"
	attributes arach "coalition station" research
	landscape land/station36
	description `This refinery was built shortly after the nearby dry dock station was finished, to ensure the system was self-sufficient to provide the hyperspace fuel needed for the visiting ships. A small fleet of tanker ships is docked on specially marked hangar bays, always ready to bring the fuel elsewhere if needed.`

> 	description `	"Ogmur" is the name of the gas giant below, named after a giant snake-like creature of their mythology. With an abnormally dense core and storms more ruthless than in any other gas giants in the Coalition, it swallowed the first five predecessors of this station before they could be completed, and though this one has remained in a stable orbit for thousands of years after its completion, superstition surrounding the planet makes it so that the Siphon receives very few visitors.`

Reviewer: ```suggestion
	description `	"Ogmur" is the name of the gas giant below, named after a giant snake-like creature of Arach mythology. With an abnormally dense core and storms more ruthless than in any other gas giants in the Coalition, it swallowed the first five predecessors of this station before they could be completed, and though this one has remained in a stable orbit for thousands of years after its completion, superstition surrounding the planet makes it so that the Siphon receives very few visitors.`
```

> 	description `	"Ogmur" is what the Arachi baptized the gas giant below, named after a giant snake-like creature of their mythology. With an abnormally dense core and storms more ruthless than in any other gas giants in the Coalition, it swallowed the first five predecessors of this station before they could be completed, and though this one has remained in a stable orbit for thousands of years after its completion, superstition surrounding the planet makes it so that the Siphon receives very few visitors.`

Reviewer 3: This description cuts it too close to the bottom of the UI. Slightly shrinking the first sentence makes it fit better.
```suggestion
	description `	"Ogmur" is the name of the gas giant below, named after a giant snake-like creature of their mythology. With an abnormally dense core and storms more ruthless than in any other gas giants in the Coalition, it swallowed the first five predecessors of this station before they could be completed, and though this one has remained in a stable orbit for thousands of years after its completion, superstition surrounding the planet makes it so that the Siphon receives very few visitors.`
```

	spaceport `The intake section has a consistently high traffic, mostly made up of Coalition merchants supplying the station and leaving with shipments of fuel to take to other Arach worlds. The science section seems to be completely closed off to non-Heliarchs. Considering that the work done here should be relatively benign, the station may also serve other purposes.`
	"required reputation" 20

planet "Periaxle Circuit"
	landscape

> 	description `The first structure of this scale the Coalition ever built, this station was designed and fully built before the three species fought the Quarg. Originally a Saryd attempt at emulating, at a much smaller scale, the Quarg's own ringworlds, almost all of the living quarters and amenities were stripped away over the course of the War of Independence, and the whole structure transformed into a gigantic particle accelerator, used by the Saryds to research weapons and shielding technology. Nowadays, the machinery remains working as it did before, but its purpose has shifted to a more general purpose research facility.`
> 	description `	First completed dozens of centuries before the war, the structure's original platings failed to last the test of millennia unscathed like the Quarg's own, and are in a constant need of replacement to prevent the miniature ring's integrity from being compromised. Every so often, a crew of maintenance craft can be seen conducting repairs on some outer section of the ring.`

Reviewer: ```suggestion
	description `The first structure of this scale the Coalition ever built, this station was designed and fully built before the War of Independence. Originally a Saryd attempt at emulating, at a much smaller scale, the Quarg's own ringworlds, almost all of the living quarters and amenities were stripped away over the course of the war, and the station transformed into a gigantic particle accelerator, used by the Saryds to research weapons and shielding technology. Nowadays, its purpose has shifted into a more general purpose research facility.`
	description `	Completed dozens of centuries before the war, the structure's original platings failed to last the test of millennia unscathed like the Quarg's own, and are in a constant need of replacement to prevent the miniature ring's integrity from being compromised. Every so often, maintenance craft can be seen conducting repairs on some outer section of the ring.`
```

	spaceport `The intake section has a consistently high traffic, mostly made up of Coalition merchants supplying the station and leaving with shipments of fuel to take to other Arach worlds. The science section seems to be completely closed off to non-Heliarchs. Considering that the work done here should be relatively benign, the station may also serve other purposes.`
	"required reputation" 20

> planet "Periaxle Circuit"
> 	landscape

Reviewer: ```suggestion
planet "Periaxle Circuit"
	attributes saryd "coalition station" research
	landscape
```

	spaceport `The intake section has a consistently high traffic, mostly made up of Coalition merchants supplying the station and leaving with shipments of fuel to take to other Arach worlds. The science section seems to be completely closed off to non-Heliarchs. Considering that the work done here should be relatively benign, the station may also serve other purposes.`
	"required reputation" 20

planet "Periaxle Circuit"
	attributes saryd "coalition station" research

> 	landscape

Reviewer: ```suggestion
	landscape land/station34
```

> 	description `The first structure of this scale the Coalition ever built, this station was designed and fully built before the War of Independence. Originally a Saryd attempt at emulating, at a much smaller scale, the Quarg's own ringworlds, almost all of the living quarters and amenities were stripped away over the course of the war, and the station was transformed into a gigantic particle accelerator used by the Saryds to research weapons and shielding technology. Nowadays, its use has shifted into a more general purpose research facility.`

Reviewer 2: ```suggestion
	description `The first structure of such scale the Coalition ever built, this station was designed and fully built before the War of Independence. Originally a Saryd attempt at emulating, at a much smaller scale, the Quarg's own ringworlds, almost every living quarter and amenities were stripped away over the course of the war in favour of a gigantic particle accelerator used by the Saryds to research weapons and shielding technology. Nowadays, its use has shifted into a more general-purpose research facility.`
```

> 	description `The first structure of this scale the Coalition ever built, this station was designed and fully built before the War of Independence. Originally a Saryd attempt at emulating, at a much smaller scale, the Quarg's own ringworlds, almost all of the living quarters and amenities were stripped away over the course of the war, and the station transformed into a gigantic particle accelerator, used by the Saryds to research weapons and shielding technology. Nowadays, its purpose has shifted into a more general purpose research facility.`

Reviewer 4: ```suggestion
	description `The first structure of this scale the Coalition ever built, this station was designed and fully built before the War of Independence. Originally a Saryd attempt at emulating, at a much smaller scale, the Quarg's own ringworlds, almost all of the living quarters and amenities were stripped away over the course of the war, and the station was transformed into a gigantic particle accelerator used by the Saryds to research weapons and shielding technology. Nowadays, its use has shifted into a more general purpose research facility.`
```

> 	description `	Completed dozens of centuries before the war, the structure's original platings failed to last the test of millennia unscathed like the Quarg's own, and are in a constant need of replacement to prevent the miniature ring's integrity from being compromised. Every so often, maintenance craft can be seen conducting repairs on some outer section of the ring.`

Reviewer 2: ```suggestion
	description `	Completed dozens of centuries before the war, the structure's original platings failed to last the test of time unscathed, unlike the Quarg's own, and are in a constant need of replacement to prevent the miniature ring's integrity from being compromised. Every so often, maintenance crafts can be seen conducting repairs on some outer section of the ring.`
```

> 	spaceport `Having very little in the form of windows to look out into the system, and not much to gaze at the small, barren planet it encompasses, the station sees very few visitors - mostly cargo hauliers bringing in supplies for the crew staffed here or the occasional university groups coming to visit the particle accelerator and deepen their knowledge.`

Reviewer 3: ```suggestion
	spaceport `Having very little in the form of windows to look out into the system, and not much to gaze at the small, barren planet it encompasses, the station sees very few visitors--mostly cargo haulers bringing in supplies for the crew staffed here or the occasional university groups coming to visit the particle accelerator and deepen their knowledge.`
```

> 	spaceport `Having very little in the form of windows to look out into the system, and not much to gaze at if looking at the small, barren planet it encompasses, the station sees very few visitors, mostly cargo haulers bringing in supplies for the crew staffed here, or university groups coming to visit the particle accelerator and deepen their knowledge.`

Reviewer 4: ```suggestion
	spaceport `Having very little in the form of windows to look out into the system, and not much to gaze at if looking at the small, barren planet it encompasses, the station sees very few visitors - mostly cargo haulers bringing in supplies for the crew staffed here, or university groups coming to visit the particle accelerator and deepen their knowledge.`
```

	spaceport `The intake section has a consistently high traffic, mostly made up of Coalition merchants supplying the station and leaving with shipments of fuel to take to other Arach worlds. The science section seems to be completely closed off to non-Heliarchs. Considering that the work done here should be relatively benign, the station may also serve other purposes.`
	"required reputation" 20

planet "Periaxle Circuit"
	attributes saryd "coalition station" research
	landscape land/station34

> 	description `The first structure of such scale the Coalition ever built, this station was designed and fully built before the War of Independence. Originally a Saryd attempt at emulating the Quarg's own ringworlds--at a much smaller scale--almost all living quarters and amenities were stripped away over the course of the war in favor of a gigantic particle accelerator used by the Saryds to research weapons and shielding technology. Nowadays, its use has shifted into a more general-purpose research facility.`

Reviewer 3: Ditto.
```suggestion
	description `The first structure of such scale the Coalition ever built, this station was designed and fully built before the War of Independence. Originally a Saryd attempt at emulating the Quarg's own ringworlds, almost all living quarters and amenities were stripped away over the course of the war in favor of a gigantic particle accelerator used by the Saryds to research weapons and shielding technology. Nowadays, its use has shifted into a more general-purpose research facility.`
```

		threshold 7000
		fleet "Large Republic" 14

> planet "Silo of Ablodab"
> 	landscape

Reviewer: ```suggestion
planet "Silo of Ablodab"
	attributes arach "coalition station" mining factory
	landscape
```

		threshold 7000
		fleet "Large Republic" 14

planet "Silo of Ablodab"
	attributes arach "coalition station" mining factory

> 	landscape

Reviewer: ```suggestion
	landscape land/station9
```

> 	description `First built by the Heliarchs just centuries after their victory over the Quarg, this station serves as both a control point for the automated strip mines on the planet below, and a smeltery to process all the raw materials brought over into the finest alloy platings. First used to produce many of the components of the two other stations in the system, the Silo now serves as a factory of replacement ship parts, to aid the local dry dock's repairing functions, if needed. Within restricted sections, under constant Heliarch patrol, the station features a few large, warehouse-like rooms, where the surplus platings are stored until they're needed.`

Reviewer 4: ```suggestion
	description `First built by the Heliarchs just centuries after their victory over the Quarg, this station serves as both a control point for the automated strip mines on the planet below, and a smeltery to process all the raw materials brought over into the finest alloy platings. First used to produce many of the components of the two other stations in the system, the Silo now serves as a factory of replacement ship parts, in order to aid the local dry dock's repairing functions if needed. Within restricted sections under constant Heliarch patrol, the station features a few large, warehouse-like rooms, where the surplus platings are stored until they're needed.`
```

> 	description `First built by the Heliarchs just centuries after their victory over the Quarg, this station serves as both a control point for the automated strip mines on the planet below, and a smeltery to process all the raw materials brought over into the finest alloy platings. First used to produce many of the components of the two other stations in the system, the Silo now serves as a factory of replacement ship parts, in order to aid the local dry dock's repairing functions if needed. Within restricted sections under constant Heliarch patrol, the station features a few large, warehouse-like rooms, where the surplus platings are stored until they're needed.`

Reviewer 2: ```suggestion
	description `First built by the Heliarchs mere centuries after their victory over the Quarg, this station serves as both a control point for the automated strip mines on the planet below and a smeltery to process all the raw materials brought up into the finest alloy platings. First used to produce many of the components of the two other stations in the system, the Silo now serves as a factory for spare ship parts in order to aid the local dry dock's repair functionalities when required. Within restricted sections under constant Heliarch patrol, the station features a few massive, warehouse-like rooms, where the surplus platings are stored.`
```

> 	spaceport `In its early days, the Silo boasted more activity than most other Coalition stations of its size, with merchant and military ships alike aiding in the construction process of its younger, sibling stations. Over thousands of years, many of the areas dedicated to living grounds and trade centers were swapped out for more and more storage sections for the alloy platings.`

Reviewer 2: ```suggestion
	spaceport `In its early days, the Silo boasted one of the highest activity as opposed to most other stations of its size, with merchant and military ships alike aiding in the construction process of its younger, sibling stations. Over thousands of years, many of the areas dedicated to living grounds and trade centres were moved out to more accessible locations across the Coalition, swapped out in favour of storage dumps to support the increasing demand for alloy platings.`
```
